internal/p2p/pex/doc.go

```diff
@@ -0,0 +1,26 @@
+/*
+Package PEX (Peer exchange) handles all the logic necessary for nodes to share information about their peers to other nodes.
```

Comment:
I typically wrap my markdown at 80 characters, but 120 also suffices.

Reply:
Do we want to standardise this? I can set my editor to 80